### DISCUSSION TITLE
question about  creating new model elements with a palette containing tools

### DISCUSSION THREAD

**Discussion by Willia-Yang** [eclipse-sirius/sirius-desktop#408] [Q&A]
I want to to create new model elements with a palette containing tool, but when I click on this tool and move the cursor to the diagram,  there is no change on the diagram, I can't create a new object on the diagram by this way, also there are no related errors in the console or .log file.

![tutieshi_640x388_2s](https://github.com/eclipse-sirius/sirius-desktop/assets/55096506/f05c2d77-ba3b-4ac4-8ac7-32283b8df8d4)

**Comment by lredor** [comment]
Same problem than https://github.com/eclipse-sirius/sirius-desktop/discussions/399
